src/components/send-message/SendMessage.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from "react";
+import "./SendMessage.css";
+import question from "../../images/question.png";
+function SendMessage() {
+  const [recipientWallet, setRecipientWallet] = useState("");
+  const [message, setMessage] = useState("");
+  const [responseAddress, setResponseAddress] = useState("");
```

Comment:
i don't think this will be necessary since the wallet approving the transaction will be the response address. we can remove this code and the associated UI code below

Reply:
I renamed it to email.